.travis.yml

```diff
@@ -88,10 +88,12 @@ install:
 script:
   - |
     if [ -z "$STACK_YAML" ]; then
-       cabal new-test
+      cabal new-test
     else
       stack build --test --no-terminal
     fi
+  - curl -o .hlint-relude.yaml https://raw.githubusercontent.com/kowainik/relude/master/.hlint.yaml
```

Comment:
It's not enough to fetch from `master` branch because `master` can be ahead of time. You should check the version we use in `summoner` and then use commit from GitHub under corresponding tag. We're using `relude-0.3.0`, it's the latest version currently. So you should take `.hlint.yaml` from the commit tagged by `v0.3.0` version:

* https://github.com/kowainik/relude/releases/tag/v0.3.0

Reply:
Right, that's a good point. I have to dig into my non-existent bash-fu for something :thinking: